screenpy/resolutions/contains_the_entry.py

```diff
@@ -19,5 +25,21 @@ class ContainsTheEntry(BaseResolution):
         )
     """
 
+    matcher: IsDictContainingEntries
     line = "a dict containing {expectation}"
     matcher_function = has_entries
+
+    @overload
+    def __init__(self, **keys_valuematchers: V) -> None:
+        ...
+
+    @overload
+    def __init__(self, keys_valuematchers: Mapping[K, V]) -> None:
+        ...
+
+    @overload
+    def __init__(self, *keys_valuematchers: Any) -> None:
+        ...
```

Comment:
@perrygoy I'm not positive this is the most correct way to hint these. Typically I try to stay away from using 'Any'.